ee/src/main/java/io/supertokens/ee/EEFeatureFlag.java

```diff
@@ -94,7 +94,7 @@ public void updateEnabledFeaturesValueReadFromDbTime(long newTime) {
     public void constructor(Main main, AppIdentifier appIdentifier) {
         this.main = main;
         this.appIdentifier = appIdentifier;
-        Cronjobs.addCronjob(main, EELicenseCheck.getInstance(main, this.appIdentifier.getAsPublicTenantIdentifier()));
+        Cronjobs.addCronjob(main, EELicenseCheck.getNewInstance(main, this.appIdentifier.getAsPublicTenantIdentifier()));
```

Comment:
Let's make this cronjob global and have it run one per app instead of one cronjob per app

Reply:
Done